narwhals/_arrow/series.py

```diff
@@ -444,6 +444,39 @@ def is_sorted(self: Self, *, descending: bool = False) -> bool:
         else:
             return pc.all(pc.less_equal(ser[:-1], ser[1:])).as_py()  # type: ignore[no-any-return]
 
+    def unique(self: Self) -> ArrowSeries:
+        pc = get_pyarrow_compute()
+        return self._from_native_series(pc.unique(self._native_series))
+
+    def sort(self: Self, *, descending: bool = False) -> ArrowSeries:
+        pc = get_pyarrow_compute()
+        series = self._native_series
+        order = "descending" if descending else "ascending"
+        sorted_indices = pc.array_sort_indices(
+            series, order=order, null_placement="at_start"
```

Comment:
with `null_placement="at_start"` I am emulating what polars does as default since we don't have such option for now

Reply:
should we add this in the code as comment? (I didn't know that this is the default in polars 😇)

Reply:
I was planning to address that to make the option available (soon...)